### PR TITLE
Cleanup module.

### PR DESCRIPTION
**Pull request by dzink**

### CODE REVIEW

**Comment by dzink** [comment]
View in BEAR at https://github.com/zivtech/bear/pull/136

File: required_for_pub.module [hunk 1]
```diff
@@ -194,136 +182,122 @@ function required_for_pub_form_validate($form, &$form_state) {
...
-  $field_string = "";
   $digit = 1;
-  if ($status == 1 && !empty($empty_fields) ) {
-    // the form is cached. Hence, the css file must be loaded again in the validate function  or  the css wonk work.
+  if (!empty($empty_fields) ) {
+    // The form is cached. Hence, the css file must be loaded again in the
+    // validate function or the css won't work.
     drupal_add_css(drupal_get_path('module', 'required_for_pub') . '/css/required_for_pub.css', array('group' => CSS_DEFAULT, 'every_page' => TRUE));
     if (module_exists('save_draft')) {
       form_set_error('required_for_pub', t('You must do "a" or "b": <p>a. Save this content as DRAFT</p> <p>b. Or PULBISH it but FILL UP "required fields for content to be published" listed below:</p>'));
```
Comment (jodyHamilton) [inline required_for_pub.module:207]: PUBLISH not PULBISH

File: required_for_pub.module [hunk 2]
```diff
@@ -194,136 +182,122 @@ function required_for_pub_form_validate($form, &$form_state) {
...
-      
       $label = $form_state['required_for_pub']['#fields_instances'][$field_name]['label'];
       form_set_error($field_name, t('%digit. %field_name', array('%field_name' => $label, '%digit' => $digit)));
       $digit = $digit + 1;
     }
-    
     return FALSE;
-    
   }
   else {
```
Comment (jodyHamilton) [inline required_for_pub.module]: I wonder if you still need this else or can just return TRUE at the end as a fallback

File: required_for_pub.module (hunk 1 above)
Comment (dzink) [inline required_for_pub.module:207]: This is fixed in https://github.com/zivtech/require-to-publish/pull/14

**Pull request merged**